tasks/main.yml

```
@@ -35,6 +35,20 @@
     state: touch
     mode: 0644
 
+- name: Ensure configuration for freshclam
+  vars:
+  - task_conf_file: "{{ freshclam_configuration_path }}"
+  - task_conf_parameters: "{{ clamav_freshclam_conf }}"
+  ansible.builtin.include_tasks: configure.yml
+  when: clamav_freshclam_conf
+
+- name: Ensure configuration for clamd
+  vars:
+  - task_conf_file: "{{ clamd_configuration_path }}"
+  - task_conf_parameters: "{{ clamav_clamd_conf }}"
+  ansible.builtin.include_tasks: configure.yml
+  when: clamav_clamd_conf
+
```

Move the task so that it comes before the `vars`.

```suggestion
- name: Ensure configuration for freshclam
  ansible.builtin.include_tasks: configure.yml
  vars:
  - task_conf_file: "{{ freshclam_configuration_path }}"
  - task_conf_parameters: "{{ clamav_freshclam_conf }}"
  when: clamav_freshclam_conf

- name: Ensure configuration for clamd
  ansible.builtin.include_tasks: configure.yml
  vars:
  - task_conf_file: "{{ clamd_configuration_path }}"
  - task_conf_parameters: "{{ clamav_clamd_conf }}"
  when: clamav_clamd_conf

```